src/server.cpp

```diff
@@ -36,7 +36,14 @@ class ServerdService : public RoutineService {
 
         m_svr.Post("/", [&](const httplib::Request& req, httplib::Response& res) {
             nlohmann::json receivedJson;
-            receivedJson = nlohmann::json::parse(req.body);
+
+            if (req.body.rfind("\"{\\",0) != 0) {
```

Comment:
```suggestion
            if (!req.body.rfind("\"{\\",0)) {
```

Ultra small nit pick